website/content/v1.7/kubernetes-guides/configuration/storage.md

```diff
@@ -96,6 +96,16 @@ First create a config patch file named `mayastor-patch.yaml` with the following
   path: /machine/nodeLabels
   value:
     openebs.io/engine: mayastor
+- op: add
+  path: /machine/kubelet/extraMounts
+  value:
+    - destination: /var/local
```

Comment:
Openebs suggests /var/local only - https://openebs.io/docs/Solutioning/openebs-on-kubernetes-platforms/talos

Reply:
I mean that mounting under different paths is an unusual thing, as usually you want kubelet to see same path structure as the host, otherwise some features won't work

Reply:
I see this one of the templates

```yaml
---
{{ if and .Values.etcd.persistence.enabled (eq .Values.etcd.persistence.storageClass "manual") }}
{{- range $index, $end := until (.Values.etcd.replicaCount | int) }}
apiVersion: v1
kind: PersistentVolume
metadata:
  name: etcd-volume-{{ $index }}
  labels:
    statefulset.kubernetes.io/pod-name: {{ print $.Release.Name }}-etcd-{{ $index }}
spec:
  storageClassName: manual
  # You must also delete the hostpath on the node
  persistentVolumeReclaimPolicy: {{ $.Values.etcd.persistence.reclaimPolicy }}
  capacity:
    storage: {{ $.Values.etcd.persistence.size | quote }}
  accessModes:
    - ReadWriteOnce
  hostPath:
    path: "/var/local/{{ $.Release.Name }}/etcd/pod-{{ $index }}"
---
{{- end }}
{{- end }}

```

Reply:
All instances (of /var/local) use `/var/local/{{ $.Release.Name }}` prefix - so i guess we can safely make it `/var/local/openebs` ?